lib/tasks/ingest.rake

```diff
@@ -486,11 +524,11 @@ namespace :ingest do
   end
 
 
- 
+
```

Comment:
Can we delete commented lines from 528 to 648?

Reply:
We agree that @nnat425 it's not doing any harm keeping it in.  I suggest keeping it since 

This code represents some of Paul's thinking from five years ago and how they addressed the ingesting then.  Here's the Git blame: https://github.com/NYPL/MyLibraryNYC/blame/master/lib/tasks/ingest.rake#L489

Judging by how long this project is living, this thinking might become relevant again if we start importing those types of CSVs again.